Documentation/getting-started.md

```diff
@@ -1,23 +1,32 @@
 ## Getting started
 
 The Adobe Experience Platform Identity for Edge Network extension has the following peer dependency, which must be installed prior to installing the identity extension:
-- [Mobile Core](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core)
+- [Mobile Core](https://github.com/adobe/aepsdk-core-android)
+
+## Configure the Identity extension in Data Collection UI
+1. Log into [Adobe Experience Platform Data Collection](https://experience.adobe.com/data-collection).
+2. From **Tags**, locate or search for your Tag mobile property.
+3. In your mobile property, select **Extensions** tab.
+4. On the **Catalog** tab, locate or search for the **Identity** extension, and select **Install**.
+5. There are no configuration settings for **Identity**.
+6. Follow the [publishing process](https://developer.adobe.com/client-sdks/documentation/getting-started/create-a-mobile-property) to update SDK configuration.
```

Comment:
Should we include this Adobe.io link?
In the developer.com, it just has "Follow the publishing process ...." with no link.

Reply:
maybe remove it here too